SourceX/dx.cpp

```diff
@@ -266,14 +266,15 @@ void RenderPresent()
 		}
 
 		// Clear buffer to avoid artifacts in case the window was resized
+#ifndef __vita__
```

Comment:
Please add a comment on why this is disabled on Vita

Reply:
Done